### ISSUE TITLE
Categories of digital IDs

### ISSUE THREAD

**Issue by UlrikeS91** [openMetadataInitiative/openMINDS_core#506]
This was raised in PR #479, in this [comment](https://github.com/openMetadataInitiative/openMINDS_core/pull/479#issuecomment-2106996138): 
"to 3) up to @apdavison ; from my side of course can this be added to whatever we see fit. Question: should we not rather introduce categories of PIDs for the different RPs/RPVs to more fluently add new PIDs when needed? @olinux would that be dynamically adopted in the KG system?"

I'm adding it as an issue to keep it open since I am about to merge the PR and this is the only open discussion point. Whether or not the ID from the PR is also added to model versions, can be discussed at a later point. If we conclude to add those categories, it can be fixed in parallel with the update.

**Comment by lzehl** [comment]
@openMetadataInitiative/openminds-developers I though about this a bit more and I'm not sure if it is worth introducing different categories of PIDs for the different research products. I would suggest we have one category for PIDs and allow them for all RPs/RPVs as selection (even if they are not used by all of them)

**Comment by lzehl** [comment]
Should be maybe discussed together with revision of publication extension (moving digital ID schemas there, making publications an other essential submodule besides core and controlledTerms).

Should also be discussed in a general revision of the digital ID schemas (rename to ID schemas ?).
- split of IRI into digital ID handle (required; +pattern; URN/IRN?) and resolvable prefix (optional; +pattern/constant; most likely URL/IRL but at least URI/IRI)
- attributes of digital ID (global, persistant, resolvable, ?)
- are version differences handled here through pattern options?

**Comment by Raphael-Gazzotti** [comment]
The different categories of digital IDs were added in https://github.com/openMetadataInitiative/openMINDS_core/pull/561, however the revision of the digital ID schemas is not in yet.

**Comment by lzehl** [comment]
I think we can close this for now and reopen a new issue when we have a plan on how to deal with digital identifiers in the future (if we want to deal with them differently than we do right now).
